packages/react-dev-utils/getProcessForPort.js

```diff
@@ -43,7 +43,7 @@ function getProcessCommand(processId, processDirectory) {
 }
 
 function getDirectoryOfProcessById(processId) {
-  return execSync('lsof -p '+ processId + ' | grep cwd | awk \'{print $9}\'', execOptions).trim();
+  return execSync('lsof -p '+ processId + ' | awk '$4=="cwd"' | awk \'{print $9}\'', execOptions).trim();
```

Comment:
The single quotes in `awk '$4=="cwd"'` need to be escaped. Currently it terminates the string, causing JS parsing to fail for this file.